stack.yaml

```diff
@@ -1,6 +1,7 @@
-resolver: nightly-2017-11-20
+resolver: lts-10.3
 packages:
 - '.'
 extra-deps:
 - pipes-http-1.0.5
 - Win32-notify-0.3.0.3
+- ansi-terminal-0.6.3.1
```

Comment:
why can't we bump this to the resolvers 0.7.1.1 ?

Reply:
I reviewed the changes and I don't see any reason we can't. I was being too conservative.